spec/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'order'
+
+describe Order do  
+
+	subject(:order) {described_class.new(menu: menu)} #injecting menu to allo method to be handled in new class menu (Order doesn't need to know!)
```

Comment:
Metrics/LineLength: Line is too long. [143/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)